fetch-in-react/my-app/src/UserDetails.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import { UserCard } from './UserCard';
+import type { User } from './Users';
+
+type Props = {
+  userId: number;
+  onCancel: () => void;
+};
+export function UserDetails({ userId, onCancel }: Props) {
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState<unknown>();
+  const [user, setUser] = useState<User>();
+
+  /* your code here (hint: useEffect) */
+
+  useEffect(() => {
+    async function load() {
+      setIsLoading(true);
+      try {
+        const response = await fetch(
+          `https://jsonplaceholder.typicode.com/users/${userId}`
+        );
+        if (!response.ok)
```

Comment:
Since this wrapped onto two lines, it's best to include the `{}`. That will make it less likely to confuse you or for you to make a mistake.